client/src/components/ideas/CreateIdea.vue

```diff
@@ -132,6 +132,8 @@ export default {
       this.tags.splice(index, 1);
     },
     submitIdea() {
+      // TODO: Remove this eslint disable once a reference to 'payload' has been resolved
+      // eslint-disable-next-line no-unused-vars
```

Comment:
Comment out the payload code, until api is done
Keep the eslint running

Reply:
Thanks. I'll do that